lib/rspec/support/ruby_features.rb

```diff
@@ -45,7 +45,7 @@ def mri?
     # Provides query methods for ruby features that differ among
     # implementations.
     module RubyFeatures
-      module_function
+    module_function
```

Comment:
Can you elaborate? It's an access modified just like `public`, `private`, `protected`. It seems a little odd here but when there are multiple modifiers it seems strange to have this indented but the others out-dented.